### DISCUSSION TITLE
Where is podman-install.zip for the UDM SE?

### DISCUSSION THREAD

**Discussion by chrisdooks** [unifi-utilities/unifi-common#419] [Q&A] [answered]
Hi. Readme states:

```
Custom Podman Builds for UDM-SE

https://github.com/unifi-utilities/unifios-utilities/actions

Extract the zip to the root of your device! Look at the assets on the Podman workflow


```

The zip file is [here](https://github.com/unifi-utilities/unifios-utilities/actions/runs/2551133100) however it's expired.

Can we have a working download please for the SE?

**Comment by jgenender** [comment]
+1 on this.  There is nowhere to get the artifact because its expired.  Can you please publish the artifact as a download so its more permanent?

**Comment by medfear** [comment]
Use the sh script in podman-update folder it wil install podman on your UDM-SE
Beware the change in build since last update of Unifi os. migration from /mnt/data to /data. Also there is a memory cap. To circumvent it you should use ssd1 or the install hard drive as your podman storage.
https://github.com/unifi-utilities/unifios-utilities/blob/main/podman-update/01-podman-update.sh

Or use this link for the old installation method
https://github.com/bdavj/unifios-utilities/actions/runs/3164252283
its a fork by bdavj on GitHub 

good luck

**Reply by jgenender** [reply]
I just thought I would give you a thank you for the above into.  I was able to install the podman and soft linked the /data directories to /mnt/data.  I also edited the /etc/containers/storage.conf and changed the graphroot setting to  "/ssd1/var/lib/containers/storage".  All works and hopefully no OOM issues.  I appreciate your added information so I didn't run out of space :-)

**Comment by jgenender** [comment]
Awesome!  Thanks!

**Comment by kb9gxk** [comment]
Any updates for Unifi-OS 3.x?

When I try to run it, I get:
`chmod: missing operand after ‘+x’
Try 'chmod --help' for more information.
mv: cannot stat '/usr/libexec/podman/conmon': No such file or directory
ln: failed to create symbolic link './conmon': File exists
chmod: missing operand after ‘+x’
Try 'chmod --help' for more information.
mv: cannot stat '/usr/bin/podman': No such file or directory
ln: failed to create symbolic link './podman': File exists
chmod: missing operand after ‘+x’
Try 'chmod --help' for more information.
mv: cannot stat '/usr/bin/runc': No such file or directory
ln: failed to create symbolic link './runc': File exists
sed: can't read /etc/containers/storage.conf: No such file or directory
sed: can't read /etc/containers/storage.conf: No such file or directory
root@ROUTER:/data/on_boot.d# 01-podman-update.sh: 48: 01-podman-update.sh: /usr/bin/podman: not found`

**Comment by KevLille** [comment]
Hi all

I have install podman on Unifi OS 3.x but I have an error with namespace mappings (don't know if it's problematic or not).

`root@UDMSE:/mnt/data/on_boot.d# podman version
WARN[0000] Failed to read current user namespace mappings
Client:       Podman Engine
Version:      4.3.0
API Version:  4.3.0
Go Version:   go1.18.3
Git Commit:   ad42af94903ce4f3c3cd0693e4e17e4286bf094b-dirty
Built:        Sun Nov 27 02:55:21 2022
OS/Arch:      linux/arm64
root@UDMSE:/mnt/data/on_boot.d# podman info
WARN[0000] Failed to read current user namespace mappings
host:
  arch: arm64
  buildahVersion: 1.28.0
  cgroupControllers:
  - cpu
  - memory
  - pids
  cgroupManager: cgroupfs
  cgroupVersion: v2
  conmon:
    package: Unknown
    path: /usr/libexec/podman/conmon
    version: 'conmon version 2.1.5, commit: '
  cpuUtilization:
    idlePercent: 80.64
    systemPercent: 7.06
`

Thanks for your help

**Reply by oyow** [reply]
Dumb question here. I have UDMSE with Unifi OS 3.x. I think the files are in the right places, but when I try to run podman ps i'm getting command not found. Obviously did something wrong

**Comment by frasderp** [comment]
Has anyone got this working on Unifi OS 3.x ? and if so... is it surviving reboots and firmware updates?

**Reply by boostchicken** [reply, marked as answer]
I dont think so.  The kernel is missing stuff.  Check the issues.
